### ISSUE TITLE
Showkase.getBrowserIntent() not generated when there are no showkase previee

### ISSUE THREAD

**Issue by matejdro** [airbnb/Showkase#334]
When there are no previews in the app, `Showkase.getBrowserIntent()` will not get generated even if there is a `@ShowkaseRoot` present.

Ideally, this Intent should still be generated, but be empty of course.

Our use case for this is that we want to include Showkase into a template project that we create other projects from (and this project includes full Showkase setup). However, because of this issue, template project will not build, because there are no Composables in it.

**Comment by vinaygaba** [comment]
Yeah I think this will help make the library be more deterministic. Especially the APIs that are generated at the top level should probably be generated to make sure there are no build failures when there are no composables.
